File: www/js/testStringBuilderFunctions.js
Bridge.define('ClientTestLibrary.TestStringBuilderFunctions', {
    statics: {
        stringBuilders: function (assert) {
            assert.expect(1);

            // TEST
            var sb = new Bridge.Text.StringBuilder("foo");

            assert.deepEqual(sb.toString(), "foo", "StringBuilder(string) .ctor");
        }
    }
});

File: tests/client/Tests/SystemFunctions/TestStringBuilderFunctions.cs
using Bridge;
using Bridge.QUnit;
using System;
using System.Text;

namespace ClientTestLibrary
{
    // Tests DateTime functions
    class TestStringBuilderFunctions
    {
        // StringBuilder functions
        public static void StringBuilders(Assert assert)
        {
            assert.Expect(1);

            // TEST
            var sb = new StringBuilder("foo");

            assert.DeepEqual(sb.ToString(), "foo", "StringBuilder(string) .ctor");
        }
    }
}
